Library/Homebrew/test/livecheck/strategy/sparkle_spec.rb

```diff
@@ -222,24 +382,25 @@
 
           "#{match[1]},#{item.version}"
         end,
-      ).to eq(["#{item_hash[0][:short_version]},#{item_hash[0][:version]}"])
+      ).to eq(["#{item_hashes[:version_123][:short_version]},#{item_hashes[:version_123][:version]}"])
 
       # Returning an array of strings from the block
       expect(
         sparkle.versions_from_content(xml[:appcast], title_regex) do |item, regex|
           [item.title[regex, 1]]
         end,
-      ).to eq([item_hash[0][:short_version]])
+      ).to eq([item_hashes[:version_123][:short_version]])
 
       expect(
         sparkle.versions_from_content(xml[:appcast], &:short_version),
-      ).to eq([item_hash[0][:short_version]])
+      ).to eq([item_hashes[:version_123][:short_version]])
 
       expect(
         sparkle.versions_from_content(xml[:appcast], title_regex) do |items, regex|
           items.map { |item| item.title[regex, 1] }
         end,
-      ).to eq(items[:appcast].map(&:short_version))
+      ).to eq(items_in_version_sort_order
+                .map(&:short_version))
```

Comment:
```suggestion
      ).to eq(items[:appcast_sorted].map(&:short_version))
```

Updating in relation to the changes above.